docs/extensibility/display-section.md

```diff
@@ -206,6 +206,7 @@ Badge widgets render texts as a status badge, using a set of predefined rules to
   If the **translations** section has a translation entry with the ID that is the same as the **placeholder** string, the translation is used.
 - **highlights** - an optional map of highlight rules. Key refers to the type of highlight, while the rule can just be a plain array of values or a string containing a [JSONata](jsonata.md) rule. Allowed keys are `informative` `positive`, `negative` and `critical`.
 - **description** - a [JSONata](jsonata.md) expression used to fetch additional information that will be displayed in a tooltip when hovering over the badge.
+- **copyable** - an optional flag, set to `true` to display a "copy to clipboard" button next to the widget.
```

Comment:
```suggestion
- **copyable** - an optional flag to display a **Copy to clipboard** button next to the widget. By default set to `true`.
```
I suggest to first describe the flag, and then inform the user what the default value is, instead of mixing the two.